frontend/src/httpClient.ts

```diff
@@ -10,6 +10,23 @@ const httpClient = axios.create({
   },
 });
 
+export type PetDetails = {
+  name: string;
+  breed: string;
+  sex: string;
+  birthDate: Date;
+  lastCheckUp: Date;
+  nextCheckUp: Date;
+};
```

Comment:
We should be more specific with the types, we might not want to send every filed to the backend.

Reply:
lastCheckUp and nextCheckUp have been deleted because they are not used even in the PetList.tsx.